readme.md

```diff
@@ -140,43 +149,43 @@ Patches represent changes.  They come with three properties:
 *   `right` ([`Node`][node], [`PropsDiff`][propsdiff], [`MoveDiff`][movediff],
     optional) — New thing
 
-#### `remove`
+#### Patch with `.type === 'remove'`
```

Comment:
@muescha could you share some more context on what's motivating this change.

`unist` uses the type attribute to represent the variant.
This is documented at https://github.com/syntax-tree/unist#nodes.

Would it be helpful to include a link to Unist to give this context?
It feels a little strange to have the `.type === 'remove'` in the title the repeat it again on the next line.

Reply:
I want to add that I’m not sure about these changes (from `` Type `Patch` `` onwards), it makes the docs too verbose for my liking.

But also: it’s a project that was never finished and has bugs, I think I’ll add that as a disclaimer somewhere at the top.

Reply:
> @muescha could you share some more context on what's motivating this change.
> 
> `unist` uses the type attribute to represent the variant.
> This is documented at [syntax-tree/unist#nodes](https://github.com/syntax-tree/unist#nodes).
> 
> Would it be helpful to include a link to Unist to give this context?

maybe - but i see only the `type` attribute here and in unist - other attributes are not the same, so a would be somehow confusing...


> It feels a little strange to have the `.type === 'remove'` in the title the repeat it again on the next line.

> I want to add that I’m not sure about these changes (from `` Type `Patch` `` onwards), it makes the docs too verbose for my liking.
> 

the headers did confusing me, so i changed it to have more clarity in the docs. a little more words remove this confusion

before i read the headers which are in code fences i was thinking:
-  they are methods of the diff tool like `diff` method?
-  or they methods of the `Patch` Type  like `Patch.remove` 

so it was not clear for me


other suggestion:


**Patch with `.type === 'remove'`**

to 

**Patch type remove**

or

**a remove Patch**

> But also: it’s a project that was never finished and has bugs, I think I’ll add that as a disclaimer somewhere at the top.

i can add also a disclaimer, but not know how is the best wording